adb_graphics/specs.py

```diff
@@ -352,7 +396,7 @@ def vort_colors(self) -> np.ndarray:
 
         grays = cm.get_cmap('Greys', 2)([0])
         ncar = cm.get_cmap(self.vspec.get('cmap'), 128) \
-                          ([15, 18, 20, 25, 50, 60, 70, 80, 85, 90, 100, 120])
+                          ([15, 18, 20, 25, 50, 60, 70, 80, 83, 90, 100, 120])
```

Comment:
I am noticing that vort_colors, vis_colors, and rh_colors are all the same code. Can we name this function something generic and just have it here once? If there are other occurrences of the same one, let's also combine those in, too.